data/json/items/gun/fishspear.json

```diff
@@ -37,6 +37,10 @@
     "skill": "pistol",
     "dispersion": 120,
     "reload": 600
+    "relative": {
```

Comment:
`proportional` is better than `relative` except when the base item might have a value of zero.
Also JSON error